src/styles/ProfilePage.css

```
@@ -1,11 +1,6 @@
-body , html {
-  height: 100%;
-}
-
 .button {
```

Lembra de Encapsular o CSS.
```suggestion
.Explorer_Page_button {
```

---

Feito